packages/react-ds/src/components/Progress/Progress.wrapper.tsx

```diff
@@ -57,7 +63,7 @@ export const ProgressWrapper = ({
 
   return (
     <div className={styleContainer} {...props}>
-      <span className={styles.value}>{selfValue}%</span>
+      <span className={styles.value}>{selfValue + modValue}%</span>
```

Comment:
Maybe we should add tests to avoid errors with float numbers?

Reply:
made on 736fd23cc910cd63aea9e47744e4c195546b946e